modules/002-deckhouse/hooks/set_module_image_value_test.go

```diff
@@ -62,7 +62,38 @@ spec:
 				Expect(f).To(ExecuteSuccessfully())
 				deployment := f.KubernetesResource("Deployment", "d8-system", "deckhouse")
 				Expect(deployment.Exists()).To(BeTrue())
-				Expect(f.ValuesGet("deckhouse.internal.currentReleaseImageName").String()).To(Equal("registry.deckhouse.io/deckhouse/ce:test"))
+				Expect(f.ValuesGet("deckhouse.internal.currentReleaseImageName").String()).To(Equal("registry.deckhouse.io/deckhouse/fe:test"))
+			})
+		})
+
+		Context("when image in absent values and regstry with port", func() {
+			BeforeEach(func() {
+				f.BindingContexts.Set(f.KubeStateSetAndWaitForBindingContexts(`
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: deckhouse
+    heritage: deckhouse
+    module: deckhouse
+  name: deckhouse
+  namespace: d8-system
+spec:
+  template:
+    spec:
+      containers:
+      - name: deckhouse
+        image: registry.deckhouse.io:666/deckhouse/ce:test666
+`, 1))
+				f.RunHook()
+			})
+
+			It("Should run", func() {
+				Expect(f).To(ExecuteSuccessfully())
+				deployment := f.KubernetesResource("Deployment", "d8-system", "deckhouse")
+				Expect(deployment.Exists()).To(BeTrue())
+				Expect(f.ValuesGet("deckhouse.internal.currentReleaseImageName").String()).To(Equal("registry.deckhouse.io/deckhouse/fe:test666"))
```

Comment:
it should be equal `registry.deckhouse.io:666/deckhouse/ce:test666` not `registry.deckhouse.io/deckhouse/fe:test666`. Or does not equal? and why?